src/main/java/uk/gov/cshr/service/AgencyTokenService.java

```diff
@@ -0,0 +1,31 @@
+package uk.gov.cshr.service;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import uk.gov.cshr.domain.AgencyToken;
+import uk.gov.cshr.service.security.IdentityService;
+
+@Service
+public class AgencyTokenService {
+
+    @Autowired
+    private IdentityService identityService;
+
+    @Autowired
+    private CsrsService csrsService;
+
+    public boolean isDomainWhiteListed(String domain) {
+        if (identityService.isWhitelistedDomain(domain)) {
+            return true;
+        }
+        return false;
```

Comment:
Can we remove redundant `if/else` and just return the result of `isWhitelistedDomain()`?

Reply:
Updated